flake8_blind_except.py

```diff
@@ -1,21 +1,41 @@
-
 try:
     import pycodestyle
 except ImportError:
     import pep8 as pycodestyle
 import re
 
-__version__ = '0.1.1'
-
-BLIND_EXCEPT_REGEX = re.compile(r'(except:)')  # noqa
+__version__ = '0.2.0'
 
+BLIND_EXCEPT_REGEX = re.compile(r'(^[ \t]*except(.*\b(Base)?Exception\b.*)?:)')  # noqa
 
 def check_blind_except(physical_line):
+    """Check for blind except statements.
+
+    >>> check_blind_except('except:')
+    (0, 'B901 blind except: statement')
```

Comment:
I like these test cases! Would you be interested in putting them in `flake8_blind_except_test.py` file that we can drive with [unittest](https://docs.python.org/3/library/unittest.html) (or some other python testing lib if you prefer?)

I can do it later if you're not interested.

Reply:
Oh sorry, I missed that we can actually run these tests with the pytest command you provided, never mind!

Reply:
The good ol' [`doctest`](https://docs.python.org/2.7/library/doctest.html) module still going strong :)

Auto tests are actually not too much work either, and on public repos you can initiate branch protection using it. If you're interested, you could add a basic CI.yml file like in e.g. https://github.com/cpcloud/ipython-autotime/pull/17